tests/functional/activitystream/test_activitystream_addon_installed.py

```diff
@@ -8,4 +8,4 @@
 class TestActivityStreamAddonInstalled(object):
 
     def test_activitystream_addon_installed(self, install_xpi, puppeteer):
-        activity_stream = ActivityStream(puppeteer)
+        activity_stream = ActivityStream(puppeteer)  # noqa
```

Comment:
I think flake8 was complainin' because `activity_stream` was unused. Added `# noqa` but not 100% certain if that is the kosherest solution.